src/main/java/core/basesyntax/dao/impl/MessageDetailsDaoImpl.java

```diff
@@ -11,11 +13,34 @@ public MessageDetailsDaoImpl(SessionFactory sessionFactory) {
 
     @Override
     public MessageDetails create(MessageDetails entity) {
-        return null;
+        Session session = null;
+        Transaction transaction = null;
+        try {
+            session = factory.openSession();
+            transaction = session.beginTransaction();
+            session.persist(entity);
+            transaction.commit();
+        } catch (Exception e) {
+            if (transaction != null) {
+                transaction.rollback();
+            }
+            throw new RuntimeException("Can't add"
+                    + " actor to DB: " + entity.getSender(), e);
```

Comment:
```suggestion
                    + "  message details to DB: " + entity.getSender(), e);
```

please check all exception messages